12-target-details.tex

```diff
@@ -140,10 +140,61 @@ \subsection{Javascript target Metatags}
 	\hline
 	\multicolumn{3}{|c|}{Javascript metatags} \\ \hline
 	Metatag &  Description & Target \\ \hline
-	@:expose \_(?Name=Class path)\_  &  Makes the class available on the \expr{window} object  & js  \\
+	@:expose \_(?Name=Class path)\_  &  Makes the class available on the \expr{window} object or \expr{exports} for node.js  & js  \\
 \end{tabular}
 \end{center}
 
+\subsection{Exposing Haxe classes for Javascript}
+\label{target-javascript-expose}
+
+It is possible to make Haxe classes or static fields available for usage in plain Javascript. 
+To expose, apply the \expr{@:expose} metatag on the desired class or static fields.
+
+This example exposes the Haxe class \ic{MyClass}.
+
+\begin{lstlisting}
+@:expose 
+class MyClass {
+    var name:String;
+    
+    function new(name:String) {
+    	this.name = name;
+    }
+    public function foo() {
+    	return 'Greetings from $name!';
+    }
+}
+\end{lstlisting}
```

Comment:
Why did you go back to inlining the code? I still prefer to have it in .hx files. :)